Makefile

```diff
@@ -17,5 +17,21 @@ reimport:
 rm:
 	docker rm -f pmm-server-tmp
 
-build-password-page:
-	cd password-page/ && npm ci && npm run build
+
+build-password-page: build-password-page-ami build-password-page-ovf
```

Comment:
what about build-password-page-docker?

Reply:
Password page is not used in Docker images